modules/logging-plugin-es-loki.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * logging/log_visualization/log-visualization-ocp-console.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-plugin-es-loki_{context}"]
+= Configuring the {log-plug} when you have the Elasticsearch log store and LokiStack installed
+
+In the {logging} version 5.8 and later, if the Elasticsearch log store is your default log store but you have also installed the LokiStack, you can enable the {log-plug} by using the following procedure.
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have installed the {clo}, the {es-op}, and the {loki-op}.
+* You have installed the {oc-first}.
+* You have created a `ClusterLogging` custom resource (CR).
+
+.Procedure
+
+. Ensure that the {log-plug} is enabled by running the following command:
++
+[source,terminal]
+----
+$ oc get consoles.operator.openshift.io cluster -o yaml |grep logging-view-plugin  \
+|| oc patch consoles.operator.openshift.io cluster  --type=merge \
+--patch '{ "spec": { "plugins": ["logging-view-plugin"]}}'
+----
+
+. Add the `.metadata.annotations.logging.openshift.io/ocp-console-migration-target: lokistack-dev` annotation to the `ClusterLogging` CR, by running the following command:
++
+[source,terminal]
+----
+$ oc patch clusterlogging instance --type=merge --patch \
+'{ "metadata": { "annotations": { "logging.openshift.io/ocp-console-migration-target": "lokistack-dev" }}}' \
+-n openshift-logging clusterlogging.logging.openshift.io/instance patched
```

Comment:
@abrennan89 the last part of this command `clusterlogging.logging.openshift.io/instance patched` is the result, so it should be added in the output section.
Currently the copy and paste of the command throws below error:
`error: there is no need to specify a resource type as a separate argument when passing arguments in resource/name form (e.g. 'oc get resource/<resource_name>' instead of 'oc get resource resource/<resource_name>'`

Reply:
Good catch, thank you!